src/VirtualClient/VirtualClient.Actions/Sysbench/SysbenchConfiguration.cs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Actions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.DependencyInjection;
+    using Polly;
+    using VirtualClient.Common;
+    using VirtualClient.Common.Extensions;
+    using VirtualClient.Common.Telemetry;
+    using VirtualClient.Contracts;
+
+    /// <summary>
+    /// Configures the MySQL database for Sysbench use.
+    /// </summary>
+    public class SysbenchConfiguration : SysbenchExecutor
+    {
+        private readonly IStateManager stateManager;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SysbenchConfiguration"/> class.
+        /// </summary>
+        /// <param name="dependencies">An enumeration of dependencies that can be used for dependency injection.</param>
+        /// <param name="parameters">A series of key value pairs that dictate runtime execution.</param>
+        public SysbenchConfiguration(IServiceCollection dependencies, IDictionary<string, IConvertible> parameters)
+            : base(dependencies, parameters)
+        {
+            this.ClientFlowRetryPolicy = Policy.Handle<Exception>().RetryAsync(3);
+            this.stateManager = this.Dependencies.GetService<IStateManager>();
+        }
+
+        /// <summary>
+        /// Disk filter specified
+        /// </summary>
+        public string DiskFilter
+        {
+            get
+            {
+                return this.Parameters.GetValue<string>(nameof(SysbenchConfiguration.DiskFilter), string.Empty);
+            }
+        }
+
+        /// <summary>
+        /// The workload option passed to Sysbench.
+        /// </summary>
+        public int TableCount
```

Comment:
Code Implementation Recommendation: Avoid significant logic in C# property getters and setters.

Move the logic here for defining the TableCount to the InitializeAsync() method to be consistent with the relative convention in other VC components.